_scss/_content.scss

```diff
@@ -13,7 +13,9 @@
  */
 
 pre {
-    background-color: #F5F8FA!important;
+    background-color: #f5f5f5!important;
```

Comment:
minor not: Should it be `#f5f8fa` ?

Reply:
oh! let me check what color was previously active. There were some rules that were duplicate, so it may be that I took the one that had the highest priority (and thus was previously active).